### PR TITLE
port word-count + include knapsack + add a function to Makefile + label hello world as warmup

### PR DESCRIPTION
**Pull request by jitwit**

### CODE REVIEW

File: code/exercises/word-count/test.ss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		     (and (= (length expected)
+			     (hashtable-size result))
+			  (fold-left (lambda (count-agrees w.c)
+				       (and count-agrees
+					    (= (cdr w.c)
+					       (hashtable-ref result
+							      (car w.c)
+							      0))))
+				     #t
+				     expected)))
```
Comment (jitwit) [inline code/exercises/word-count/test.ss]: Actually, this requires students to implement the solution using hashtables. I should redo the equality test to allow for association lists, too.

File: Makefile [hunk 2]
```diff
@@ -48,15 +50,15 @@ bin/configlet :
 
 # configuration
 config.json : code/config.ss
-	echo "(make-config)" | $(chez) -q $(load)
+	$(call exercise, "(make-config)")
```
Comment (guygastineau) [inline Makefile]: These are nice improvements, but they probably belong in a separate PR

File: code/config.ss [hunk 3]
```diff
@@ -19,7 +19,7 @@
       (auto_approve . #t)
       (unlocked-by)
       (difficulty . 1)
-      (topics))
+      (topics warmup))
```
Comment (guygastineau) [inline code/config.ss]: Same thing with changing these values in this PR.

**Comment by guygastineau** [comment]
Are you ready to merge this?  I certainly would prefer if the PR got any bigger.  It has my approval.

**Comment by jitwit** [comment]
Just cleaned up the example solution a bit, will do

**Comment by guygastineau** [comment]
BTW, I will start implementing some exercises (like bob) tonight.

**Comment by jitwit** [comment]
Sounds good!

File: code/exercises/knapsack/knapsack.scm [hunk 4]
```diff
@@ -0,0 +1,7 @@
+(import (rnrs (6)))
+
+(load "test.scm")
+
+(define (knapsack capacity weights vlues)
```
Comment (jitwit) [inline code/exercises/knapsack/knapsack.scm]: ugh

Comment (jitwit) [reply code/exercises/knapsack/knapsack.scm]: I was thinking of adding a linting pass making sure each function in the student sub is in the example solution and that the arguments match

Comment (guygastineau) [reply code/exercises/knapsack/knapsack.scm]: Sure, that sounds pretty cool.
